src/hb-uniscribe.cc

```diff
@@ -603,6 +605,56 @@ hb_uniscribe_font_get_hfont (hb_font_t *font)
   return data ? data->hfont : nullptr;
 }
 
+static hb_blob_t *
+_hb_uniscribe_hdc_reference_table (hb_face_t *face HB_UNUSED, hb_tag_t tag, void *user_data)
+{
+  HDC hdc = (HDC) user_data;
+
+  char *buffer = NULL;
+  DWORD length = 0;
+
+  length = GetFontData (hdc, hb_uint32_swap (tag), 0, buffer, length);
+  if (unlikely (length == GDI_ERROR)) return hb_blob_get_empty ();
+
+  buffer = (char *) malloc (length);
+  length = GetFontData (hdc, hb_uint32_swap (tag), 0, buffer, length);
+  if (unlikely (length == GDI_ERROR)) length = 0;
+
+  return hb_blob_create ((const char *) buffer, length, HB_MEMORY_MODE_READONLY, buffer, free);
+}
+
+/**
+ * hb_uniscribe_hfont_face_create:
+ * @hdc: a HFONT object.
+ *
+ * Return value: #hb_face_t object corresponding to the given input
+ *
+ * Since: REPLACEME
+ **/
+hb_face_t *
+hb_uniscribe_hfont_face_create (HFONT hfont)
+{
+  // XXX: Release things...
+  HDC hdc = GetDC(nullptr);
+  SelectObject(hdc, hfont);
```

Comment:
What about moving these two lines to `_hb_uniscribe_hdc_reference_table` (and rename it as well)?

Reply:
Done